-------------------------------------------------------------------------------
Test set: open.testing.suite.unittest.demo.AddFunTest
-------------------------------------------------------------------------------
Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.075 s <<< FAILURE! - in open.testing.suite.unittest.demo.AddFunTest
testAddThree  Time elapsed: 0 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <140> but was: <20>
	at open.testing.suite.unittest.demo.AddFunTest.testAddThree(AddFunTest.java:39)

